packages/odf/modals/configure-performance/configure-performance-modal.tsx

```diff
@@ -183,13 +189,15 @@ const ConfigurePerformanceModal: React.FC<ConfigurePerformanceModalProps> = ({
       width={950}
     >
       <ModalBody className="configure-performance-modal--overflow">
-        <ConfigurePerformance
-          onResourceProfileChange={onProfileChange}
-          resourceProfile={resourceProfile}
-          profileRequirementsText={ProfileRequirementsModalText}
-          selectedNodes={selectedNodes}
-          osdAmount={osdAmount}
-        />
+        {isProviderMode && (
```

Comment:
This should be negated ?
```suggestion
        {!isProviderMode && (
```